src/App.css

```diff
@@ -1,33 +1,23 @@
 .App {
   text-align: center;
+  background-color: #ececec;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+html {
+  position: relative;
+  min-height: 100%;
+  background-color: #ececec;
```

Comment:
could you add variable for this color? It's the same in line 14. so you can reuse it